scripts/src/primazatest/runtest.py

```diff
@@ -1005,10 +1005,12 @@ def main():
                         dest="input_dir",
                         help="directory for kubeconfigs used for user tests",
                         required=False)
-    parser.add_argument("-j", "--sa_namespace",
-                        dest="sa_namespace",
-                        help="directory for kubeconfigs used for user tests",
-                        required=False)
+    parser.add_argument("-j", "--service_account_namespace",
```

Comment:
Need to now use this to pass as a parameter in one of the tests, so tests include running with a named namespace.
Also makefile needs to be updated to pass the parameter to runtest.

Reply:
shouldn't it be `--service-account-namespace`?

Reply:
yes good catch on this, updated it